theano/gof/op.py

```diff
@@ -966,7 +966,7 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         if new_node is not None:
             node = new_node
 
-        if self._op_use_c_code:
```

Comment:
We may need to talk about that IRL, because I don't get what the issue is.
`self._op_use_c_code` is set in `Op.__new__`, so that, as the comment says, "all existing Ops get a _op_use_c_code attribute".
So, the only way for an unpickled Op instance to _not_ have `_op_use_c_code` would be if there was a `__getstate__` implemented for that Op subclass that would get rid of that attribute, since that is not what the Op base class does.
My point is that, in the case where `__getstate__` is implemented, it is reasonable to add a `__setstate__` for that specific subclass. Moreover, getting rid of `_op_use_c_code` is a symptom that the `__getstate__` was not well-written, and we should address that rather than mask it. That is what we did for Gemm and related classes.

Do you know which other Op was missing `_op_use_c_code` after unpickling?